src/MoBi.Core/Reporting/ChartsReporter.cs

```diff
@@ -1,28 +1,26 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using OSPSuite.TeXReporting.Items;
 using OSPSuite.Core.Chart;
 using OSPSuite.Infrastructure.Reporting;
+using OSPSuite.TeXReporting.Items;
 
 namespace MoBi.Core.Reporting
 {
-   internal class ChartsReporter : OSPSuiteTeXReporter<IEnumerable<ICurveChart>>
+   internal class ChartsReporter : OSPSuiteTeXReporter<IReadOnlyCollection<CurveChart>>
```

Comment:
Did you check that this is working? If what's passed to the Reporter is an actual IEnumerable, this TeXReporter won't be found